lib/routes/account.js

```diff
@@ -37,6 +37,8 @@ module.exports = function (
     )
   ]
 
+  var verificationReminder = require('../verification-reminders')(log, db)
```

Comment:
Only motivation would be for consistency, which isn't as consistent as I thought it was. This is safe to ignore.